xpano/gui/panels/bugreport_pane.cc

```diff
@@ -0,0 +1,78 @@
+#include "xpano/gui/panels/bugreport_pane.h"
+#include <iostream>
```

Comment:
In general we want to include only the headers we actually use in the code./misc. (It is a bit cumbersome to find out which headers to add/remove each time, but there is a tool for that: https://github.com/include-what-you-use/include-what-you-use)

In this case you can remove a quite a few headers;

```
C:/Users/tomas/Documents/dev/poshi/xpano/gui/panels/bugreport_pane.cc should remove these lines:
- #include <algorithm>  // lines 4-4
- #include <iostream>  // lines 2-2
- #include <iterator>  // lines 5-5
- #include <sstream>  // lines 6-6
- #include <utility>  // lines 8-8
- #include <vector>  // lines 9-9
- #include "xpano/utils/future.h"  // lines 14-14
- #include "xpano/utils/text.h"  // lines 15-15
```

Also `#include "xpano/utils/sdl_.h` will likely be removed

Reply:
Yes I had missed that, my bad.

Reply:
No problem, I also need to put this in the contribution guidelines